mkdocs.yml

```diff
@@ -73,7 +73,62 @@ plugins:
   - redirects:
       redirect_maps:
         'tutorial/index.md': 'guides/pet-shop/index.md'
-        'tutorial/pet-shop/index.md': 'guides/pet-shop/index.md'          
+        'tutorial/pet-shop/index.md': 'guides/pet-shop/index.md'
         'tutorials/index.md': 'guides/pet-shop/index.md'
         'tutorials/pet-shop/index.md': 'guides/pet-shop/index.md'
         'docs/ganache/overview/index.md': 'docs/ganache/index.md'
+#       Truffle docs restructure
+        'docs/truffle/getting-started/compiling-contracts.md': 'docs/truffle/how-to/compile-contracts.md'
+        'docs/truffle/getting-started/creating-a-project.md': 'docs/truffle/how-to/create-a-project.md'
+        'docs/truffle/getting-started/installation.md': 'docs/truffle/how-to/install.md'
+        'docs/truffle/getting-started/interacting-with-your-contracts.md': 'docs/truffle/how-to/contracts/interact-with-your-contracts.md'
+        'docs/truffle/getting-started/package-management-via-ethpm.md': 'docs/truffle/how-to/package-management-via-npm.md'
+        'docs/truffle/getting-started/package-management-via-npm.md': 'docs/truffle/how-to/package-management-via-npm.md'
+        'docs/truffle/getting-started/preserving-files-and-content-to-storage-platforms.md': 'docs/truffle/how-to/preserve-files-and-content-to-storage-platforms.md'
+        'docs/truffle/getting-started/running-migrations.md': 'docs/truffle/how-to/contracts/run-migrations.md'
+        'docs/truffle/getting-started/truffle-with-metamask.md': 'docs/truffle/how-to/truffle-with-metamask.md'
+        'docs/truffle/getting-started/using-the-build-pipeline.md': 'docs/truffle/how-to/use-the-build-pipeline.md'
+        'docs/truffle/getting-started/using-the-truffle-dashboard.md': 'docs/truffle/how-to/use-the-truffle-dashboard.md'
+        'docs/truffle/getting-started/using-the-truffle-debugger.md': 'docs/truffle/how-to/debug-test/use-the-truffle-debugger.md'
+        'docs/truffle/getting-started/using-truffle-develop-and-the-console.md': 'docs/truffle/how-to/debug-test/use-truffle-develop-and-the-console.md'
+        'docs/truffle/testing/testing-your-contracts.md': 'docs/truffle/how-to/debug-test/test-your-contracts.md'
+        'docs/truffle/testing/writing-tests-in-javascript.md': 'docs/truffle/how-to/debug-test/write-tests-in-javascript.md'
+        'docs/truffle/testing/writing-tests-in-solidity.md': 'docs/truffle/how-to/debug-test/write-tests-in-solidity.md'
+        'docs/truffle/getting-started/writing-external-scripts.md': 'docs/truffle/how-to/write-external-scripts.md'
+        'docs/truffle/distributed-ledger-support/working-with-hyperledger-evm.md': 'docs/truffle/how-to/distributed-ledger-support/hyperledger-evm.md'
+        'docs/truffle/distributed-ledger-support/working-with-quorum.md': 'docs/truffle/how-to/distributed-ledger-support/quorum.md'
+        'docs/truffle/contact-the-developers.md': 'docs/truffle/contact-the-developers.md'
```

Comment:
Just spotted that this is referencing itself causing redirect loop

Reply:
Good spot! I was wondering why the page wasn't loading.